Azure/azure-cli-extensions
# [Diskpool] `az disk-pool`: Update version to stable swagger, add commands `az disk-pool list-zones`, `az disk-pool redeploy`

https://github.com/Azure/azure-cli/issues/19470, update version to stable swagger.
Add `az disk-pool list-zones`
Add `az disk-pool redeploy`

To test run:
1. `az disk-pool list-zones -l {region}`
2. `az disk-pool redeploy --name {diskPoolName} --resource-group {rg}`
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines


For new extensions:



### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.
